lib/hanami/slice_registrar.rb

```diff
@@ -118,6 +122,15 @@ def load_slice(slice_name)
       register(slice_name, slice_class)
     end
 
+    def find_slice_require_path(slice_name)
```

Comment:
A doc comment about this would be good too. Explain the algorithm and the order of preference for slice file lookup.